src/audio/volume/volume.c

```diff
@@ -229,21 +229,15 @@ static struct comp_dev *volume_new(struct sof_ipc_comp *comp)
 		if (vol->min_value < VOL_MIN) {
 			/* Use VOL_MIN instead, no need to stop new(). */
 			cd->vol_min = VOL_MIN;
-			trace_volume_error_with_ids(dev,
-						    "volume_new(): "
-						    "vol->min_value was "
-						    "limited to VOL_MIN.");
+			comp_err(dev, "volume_new(): vol->min_value was limited to VOL_MIN.");
```

Comment:
I'm not sure this should even be an error in the first place (mentioning in passing). There was a discussion about this but I forgot what the conclusion was.

Reply:
Yes, noticed that one too, could be a warning once there is one for that category. Please note I did not try to fix things (there are even some dangerous uses in some places I spotted while verifying the conversion) since should come as a separate patch I think.

Reply:
True, this should only actually focus on the trace macros themselves, a follow-up is pretty good to think about though.